articles/article-093-rices-theorem.md

```diff
@@ -0,0 +1,40 @@
+<!-- BEGIN TITLE -->
+# Why There Is No Perfect Anti-Virus
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In computer science, simple "Yes/No" questions are called **decision
+problems**.  A decision problem is called **decidable** if we can
+write a program that can always find an answer for such a problem.
+
+An example for such a decision problem is "does this program contain a
```

Comment:
I would add a question mark to the "does this program contain a virus?", for clarity

Reply:
Agreed